tests/TestRvsdgs.cpp

```diff
@@ -817,10 +818,9 @@ IndirectCallTest1::SetupRvsdg()
   };
 
   auto SetupTestFunction = [&](
-    lambda::output * fctindcall,
-    lambda::output * fctthree,
-    lambda::output * fctfour)
-  {
+    lambda::output *fctindcall,
+    lambda::output *fctthree,
+    lambda::output *fctfour) {
```

Comment:
Same here

Reply:
Overhauled the TestRvsdgs.hpp and TestRvsdgs.cpp